modules/test/playwright/pages/object-web/ModelBuilderPage.ts

```diff
@@ -42,6 +42,15 @@ export class ModelBuilderPage {
 	readonly page: Page;
 	readonly postalAddressObjectRelationshipWarning: Locator;
 	readonly rightSidebar: Locator;
+	readonly rightSidebarObjectDefinitionActivateObject: Locator;
+	readonly rightSidebarObjectDefinitionEntryTitleField: Locator;
+	readonly rightSidebarObjectDefinitionLabel: Locator;
+	readonly rightSidebarObjectDefinitionLabelLocalizationButton: Locator;
+	readonly rightSidebarObjectDefinitionPanelLink: Locator;
+	readonly rightSidebarObjectDefinitionPluralLabel: Locator;
+	readonly rightSidebarObjectDefinitionPluralLabelLocalizationButton: Locator;
+	t;
+	readonly rightSidebarObjectDefinitionScope: Locator;
```

Comment:
i'm not going to ask to change this now, but i think we need to do a refactor of this file; there are a lot of locators here already and it's been difficult to manage them in size and in how we name them.

maybe its time to separate the modals and the sidebars into other pages like we did on forms and picklist for example, we can create a ticket for this and do asap; if you have any other suggestions please let me know!

Reply:
I completely agree, I had that feeling too.

As I work on the next tests, I will try to separate these locators into smaller files!